hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/impl/ERCTransferPrecompile.java

```diff
@@ -221,18 +226,22 @@ public static List<TokenTransferWrapper> decodeERCTransfer(
         addSignedAdjustment(fungibleTransfers, token, recipient, amount.longValueExact());
         addSignedAdjustment(fungibleTransfers, token, caller, -amount.longValueExact());
 
-        return Collections.singletonList(
-                new TokenTransferWrapper(NO_NFT_EXCHANGES, fungibleTransfers));
+        final List<TokenTransferWrapper> tokenTransferWrappers =
```

Comment:
nit: can use `var` here and in the rest repeated places

Reply:
done